--- 1.5.1/src/scanbuttond/backends/hp5590.c.orig	2018-05-01 19:36:24.037016396 +0200
+++ 1.5.1/src/scanbuttond/backends/hp5590.c	2018-05-01 19:36:49.172968924 +0200
@@ -268,17 +268,17 @@
 /* Structure describing control URB */
 struct usb_in_usb_ctrl_setup
 {
-       u_int8_t  bRequestType;
-       u_int8_t  bRequest;
-       u_int16_t wValue;
-       u_int16_t wIndex;
-       u_int16_t wLength;
+       uint8_t  bRequestType;
+       uint8_t  bRequest;
+       uint16_t wValue;
+       uint16_t wIndex;
+       uint16_t wLength;
 } __attribute__ ((packed));
 
 static int
 hp5590_get_ack (scanner_t *scanner)
 {
-       u_int8_t        status;
+       uint8_t        status;
        int                     ret;
 
        /* Check if USB-in-USB operation was accepted */
@@ -312,8 +312,8 @@
        int                                                             ret;
        unsigned int                                    len;
        unsigned char                                   *ptr;
-       u_int8_t                                                ack;
-       u_int8_t                                                response;
+       uint8_t                                                ack;
+       uint8_t                                                response;
 
        /* IN (read) operation will be performed */
        if (requesttype & USB_DIR_IN)
@@ -478,7 +478,7 @@
 static int
 hp5590_verify_last_cmd (scanner_t *scanner, unsigned int cmd)
 {
-       u_int16_t               verify_cmd;
+       uint16_t               verify_cmd;
        unsigned int    last_cmd;
        unsigned int    core_status;
        int                             ret;
@@ -534,7 +534,7 @@
 scanbtnd_get_button(scanner_t* scanner)
 {
        int             button = 0;
-       u_int16_t       button_status;
+       uint16_t       button_status;
        int                     ret;
 
        if (!scanner->is_open)
